src/stream/src/executor/mod.rs

```diff
@@ -231,6 +233,15 @@ impl Barrier {
         }
     }
 
+    pub fn new_test_barrier_with_prev_epoch(epoch: u64, prev_epoch: u64) -> Self {
```

Comment:
How about `with_prev_epoch_for_test`?

Reply:
> How about `with_prev_epoch_for_test`?

this is [wangrunji0408](https://github.com/wangrunji0408)'s suggestion. Anyway I'm to alter it.

Reply:
> How about `with_prev_epoch_for_test`?

`new_test_barrier_with_prev_epoch` is [wangrunji0408](https://github.com/wangrunji0408)'s suggestion. Anyway I'm to alter it.